vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -40,7 +44,7 @@ def save_inventory(type, parent, hash)
     found = parent.send(type)
     if found.nil?
       found = parent.send("build_#{type}", hash.except(:id))
-      parent.send("#{type}=", found)
+      parent.update_attributes!(type => found)
```

Comment:
So, I'm testing this piece out in particular, and I noticed 2 things.

1) The old line `parent.send("#{type}=", found)` didn't do anything at all from what I can tell, so there was no need for it.
2) When you save the parent (which we always do), the built record gets saved.  This happens regardless of autosave.

So, the line in 1 can definitely go, but I don't understand why you need the `update_attributes!`.  What specific issue does this solve, because I can't create a pattern that fails locally the "old" way?